pkg/apis/serving/v1alpha1/route_conversion_test.go

```diff
@@ -216,6 +254,9 @@ func TestRouteConversion(t *testing.T) {
 		// A variant of the test that uses `name:`,
 		// but end up with what we have above anyways.
 		t.Run(test.name+" (deprecated)", func(t *testing.T) {
+			if test.wantErr {
+				t.Skip("skipping error rows")
```

Comment:
why?

Reply:
because the errors largely center around both forms existing :)